scripts/deploy-shielder.sh

```diff
@@ -46,10 +46,17 @@ if [ -n "${AR_PUBLIC_KEY:-}" ]; then
 elif [ -n "${AR_SEED:-}" ]; then
     # Generate the public key using ar-cli
     echo "Generating AR key pair from seed..."
-    cargo run --bin ar-cli -- generate --seed ${AR_SEED} --endianess big-endian
+    cargo run --bin ar-cli -- generate --seed ${AR_SEED}
+    AR_PUBLIC_KEY=$(cat public_key.bin | cast from-bin)
+    AR_PUBLIC_KEY_X="0x$(echo $AR_PUBLIC_KEY | cut -c3-66)"
+    AR_PUBLIC_KEY_Y="0x$(echo $AR_PUBLIC_KEY | cut -c67-130)"
+    
+    # Assert correct length of AR_PUBLIC_KEY_X and AR_PUBLIC_KEY_Y
+    if [ ${#AR_PUBLIC_KEY_X} -ne 66 ] || [ ${#AR_PUBLIC_KEY_Y} -ne 66 ]; then
```

Comment:
why 66?

Reply:
0x is there

Reply:
And 64 hex symbols is 32 bytes, which represent a single U256 in big endian.